index.js

```diff
@@ -458,6 +466,18 @@ module.exports = {
     'operator-linebreak': 'warn',
     // enforce padding within blocks
     'padded-blocks': ['warn', 'never'],
+    // require or disallow padding lines between statements
+    'padding-line-between-statements': ['warn',
+      { blankLine: 'always', prev: '*', next: 'return' },
+      { blankLine: 'always', prev: ['const', 'let', 'var'], next: '*' },
+      {
+        blankLine: 'any',
+        prev: ['const', 'let', 'var'],
+        next: ['const', 'let', 'var']
+      },
+      { blankLine: 'always', prev: 'directive', next: '*' },
+      { blankLine: 'any', prev: 'directive', next: 'directive' }
+    ],
```

Comment:
It looks like this rule replaces a few other now-deprecated rules.  I don't see those being removed from this configuration, but they should be, because otherwise we'll get warnings when running eslint against our projects.